data/wicked/check_macvtap.c

```diff
@@ -45,6 +46,24 @@ char fn[FILENAME_LENGTH];
 // Read buffer
 unsigned char buffer[PACKET_SIZE];
 
+void read_macaddr() {
```

Comment:
isn't there something in stdlib knowing how to read MAC ? or in kernel syscalls ? looks like we inventing the wheel here ...

Reply:
I guess there might be something in netlink, but I expect it is much more difficult then asking sysfs.